### DISCUSSION TITLE
Scroll by line not available in epub?

### DISCUSSION THREAD

**Discussion by appyface** [sumatrapdfreader/sumatrapdf#2434] [General]
Hello,
I am not able to scroll by line in epub. Up/down arrow and j/k scroll only by page? 
PDFs scroll by line using up/down and j/k. 
Is there a setting I overlooked somewhere?
Thank you.

**Comment by GitHubRulesOK** [comment]
Use latest Pre-release for testing https://www.sumatrapdfreader.org/prerelease
Where the PDF and ePub behaviors should be roughly the same since using the same interface
 whereas older versions had two different interfaces thus different behaviors.

**Reply by appyface** [reply]
I will do. Thank you!

**Reply by appyface** [reply]
Works! Thanks again.
